nablapps/interactive/templates/interactive/code_golf.html

```diff
@@ -79,12 +82,15 @@ <h4 class="alert-heading">Output:</h4>
              status: null,  // One of "correct", "incorrect", "spinning"
              user_code: null,
              output: null,
+             longerThanBest: null
```

Comment:
```suggestion
             longerThanBest: false
```

I think false is a more natural default here